# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/fetch/http-cache/304-update.any.js.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/fetch/http-cache/304-update.any.js.ini
--- a/testing/web-platform/meta/fetch/http-cache/304-update.any.js.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/fetch/http-cache/304-update.any.js.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -6,8 +6,8 @@
 [304-update.any.worker.html]
   [HTTP cache updates stored headers from a Last-Modified 304]
     expected:
+      if (os == "linux") and debug and webrender and not fission and not swgl: ["FAIL", "PASS"]
       if (os == "win") and not debug and webrender and fission: PASS
-      if (os == "android") and debug and not webrender: ["FAIL", "PASS"]
       if (os == "win") and not debug and not webrender: PASS
       if (os == "win") and debug: PASS
       [PASS, FAIL]
@@ -23,8 +23,8 @@
 [304-update.any.sharedworker.html]
   [HTTP cache updates stored headers from a Last-Modified 304]
     expected:
-      if (os == "linux") and not fission and (processor == "x86_64") and not swgl and debug and webrender: ["FAIL", "PASS"]
-      if (os == "win") and not webrender: PASS
+      if (os == "win") and not webrender and not debug and (processor == "x86"): PASS
+      if (os == "win") and not webrender and debug: PASS
       if (os == "mac") and not debug: PASS
       [PASS, FAIL]
 